src/pages/jaehyun/Login/JaehyunLogin.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+import './JaehyunLogin.scss';
+import InputBox from './InputBox/InputBox';
+
+class JaehyunLogin extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      userName: '',
+      password: '',
+      userIdCheck: true,
+      passwordCheck: true,
+    };
+  }
+
+  // inputChangeHandler = (name, value, disable) => {
+  //   this.setState({
+  //     [name]: value,
+  //     [name === 'userName' ? 'userIdCheck' : 'passwordCheck']: disable,
+  //   });
+  // };
+
+  inputChangeHandler = e => {
+    const { name, value } = e.target;
+    if (name === 'userName') {
+      this.setState({
+        userName: value,
+        userIdCheck: value.indexOf('@') === -1,
+      });
+    }
+    if (name === 'password') {
+      this.setState({
+        password: value,
+        passwordCheck: value.length < 5,
+      });
+    }
```

Comment:
```suggestion
    const { name, value } = e.target;
    this.setState({[name]:value});
    }
```
계산된 속성명 사용해서 줄일 수 있어보입니다, passwordCheck, userIdCheck는 필요없는 값이니까 없애도 되보입니다.
그리고 check는 뭔가 체크박스가 체크되고 안되고 같지 않나요? isPasswordValid 등의 의미가 드러나고, boolean임을 알아볼 수 있는 네이밍을 해주세요